lib/utils.pm

```diff
@@ -412,6 +413,24 @@ sub sle_version_at_least {
     die "unsupported SLE VERSION $version in check";
 }
 
+sub sle_hddversion_at_least {
```

Comment:
please refactor that. this function and sle_version_at_least are about the same, just the variable checked is differently. So instead of having one more function duplicating this, extend sle_version_at_least so you can call it like:

```
sle_version_at_least '12-SP2, version_variable => 'HDDVERSION'